iOS/src/toga_iOS/screen.py

```diff
@@ -0,0 +1,27 @@
+from toga.screen import Screen as ScreenInterface
+
+
+class Screen:
+    _instances = {}
+
+    def __new__(cls, native):
+        if native in cls._instances:
+            return cls._instances[native]
+        else:
+            instance = super().__new__(cls)
+            instance.interface = ScreenInterface(_impl=instance)
+            instance.native = native
+            cls._instances[native] = instance
+            return instance
+
+    def get_name(self):
+        return "IOS Screen"
```

Comment:
Apple has very specific naming guidelines; it's always "iOS".
```suggestion
        return "iOS Screen"
```